vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -329,16 +329,26 @@
 
   private static void handleException(HttpServerExchange exchange) {
     final var e = (Exception) exchange.getAttachment(ExceptionHandler.THROWABLE);
-    if (e instanceof ValidationException) {
-      exchange.setStatusCode(StatusCodes.BAD_REQUEST);
-    } else {
-      exchange.setStatusCode(StatusCodes.INTERNAL_SERVER_ERROR);
-    }
     e.printStackTrace();
+    exchange.setStatusCode(StatusCodes.INTERNAL_SERVER_ERROR);
     exchange.getResponseHeaders().put(Headers.CONTENT_TYPE, CONTENT_TYPE_TEXT);
     exchange.getResponseSender().send(e.getMessage());
   }
 
+  private static void handleValidationErrors(HttpServerExchange exchange, String message) {
+    exchange.setStatusCode(StatusCodes.BAD_REQUEST);
+    System.out.println(message);
+    exchange.getResponseHeaders().put(Headers.CONTENT_TYPE, CONTENT_TYPE_TEXT);
+    exchange.getResponseSender().send(message);
+    return;
+  }
+
+  private static void handleValidationErrors(
+      HttpServerExchange exchange, List<ValidationError> errors) {
+    String message = errors.stream().map(e -> e.toString()).collect(Collectors.joining("  "));
```

Comment:
It's not needed for a particular application. ValidationError.toString() returns an error in the form "property: message." (including the period). Since it's joining together a list of error messages, I figured a space-separated list of errors would be easier to read than e.g. a newline-separated list of errors.